gammapy/spectrum/tests/test_spectrum_extraction.py

```diff
@@ -84,6 +84,19 @@ def test_spectrum_extraction(pars,results,tmpdir):
     assert_allclose(ana.observations[0].lo_threshold,results['ethresh'])
 
     assert_quantity_allclose(obs23523.aeff.evaluate(energy=5*u.TeV),results['aeff'])
+    
+    # Write on set of output files to gammapy-extra as input for other tests
+    # and check I/O
+    if not pars['containment_correction']:
```

Comment:
I merge this, please leave a note if you disagree with the code above, so we can adapt it